02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>same comment about the image, I'm not sure if I can see the thing that I am supposed to see,</p><p>it's a bit confusing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>Replaced this with an image, along with a note callout that further explains the difference between direct vs. heap memory in Java.</p>